other/upgrade_2-1_postgresql.sql

```diff
@@ -2055,15 +2055,19 @@ CREATE INDEX {$db_prefix}ban_items_id_ban_ip ON {$db_prefix}ban_items (ip_low,ip
 --- helper function for ip convert
 /******************************************************************************/
 ---# the function migrate_inet
-CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
-AS
-$$
-BEGIN
-   RETURN (trim(val))::inet;
-EXCEPTION
-   WHEN OTHERS THEN RETURN NULL;
-END;
-$$ LANGUAGE plpgsql;
+---{
+upgrade_query("
+	CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
+	AS
+	$$
+	BEGIN
+	   RETURN (trim(val))::inet;
+	EXCEPTION
+	   WHEN OTHERS THEN RETURN NULL;
+	END;
+	$$ LANGUAGE plpgsql;"
+);
+---}
```

Comment:
So i checked to  day the code and i can't understand fully why this happen.
Because the query check should be already disable by the upgrade_query function and
when this plpgsql function already exists should this replaced by this function (CREATE OR REPLACE FUNCTION migrate_inet) because of the replace in the statement.